### DISCUSSION TITLE
Marker bei einem Benutzerdefinierten Werkzeug anzeigen

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#78] [Ideas]
Wäre es möglich bei einem Benutzerdefinierten Werkzeug, welches nur durch einen Klick in die Karte ausgelöst wird (z.B. Sonnenstandsabfrage), einen Marker in die Karte zu zeichnen? 
Dies wäre vom Vorteil, damit der Benutzer weiß, wo er die Abfrage gestartet hat. 

Beim Höhenprofil sieht man beispielsweise den Verlauf der Linie.

**Comment by faulgurke** [comment]
Gute Idee - ich unterstütze diesen Wunsch.

**Comment by gis-eni** [comment]
Kommt in nächsten Build >=5.23.260x:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/a4ef6be7-3a2f-4538-b4b6-810fe4460ca4)
